applications/projects/working-with-projects.adoc

```diff
@@ -37,26 +37,53 @@ In {product-title} clusters that use the Customer Cloud Subscription (CCS) model
 ====
 endif::openshift-dedicated[]
 
-include::modules/creating-a-project-using-the-web-console.adoc[leveloffset=+1]
+[id="working-with-projects-create-project"]
+== Creating a project
 
-include::modules/odc-creating-projects-using-developer-perspective.adoc[leveloffset=+1]
+You can use the {product-title} web console or the OpenShift CLI (oc) to create a project in your cluster.
```

Comment:
`oc` is not properly formatted as monospace here. `{oc-first}` can be used instead.